### ISSUE TITLE
Pressing DEL in Global Search should not delete note

### ISSUE THREAD

**Issue by Redsandro** [Zettlr/Zettlr#3493]
### Description

There are two bugs that, when in a hurry, mixed with some healthy stupidity, can be dangerous, as I have personally experienced.

The _Search Terms_ box is focused when opening the _Global Search_. When pressing `TAB`, you expect the next field to be focused and the contents to be selected, ready to be removed by pressing `DEL`.

The first problem is that `TAB` doesn't focus the second field, but selects something else first. (What exactly?) To focus the next field, you need to press `TAB` two times.

The second problem is that `DEL` opens the Delete File dialog. I'm filling in a search form - not browsing a file list - and `DEL` is a legit search form related button here. It shouldn't toggle between deleting search terms and whole files, depending on what search form element is selected.

In my case, I often want the Restrict Search box to be empty. So I automatically press `TAB` `DEL` `ENTER` to empty the Restrict Search to directory box. Wait, what was that dialog? (Try again) NOOOO!! `DEL` asks you to delete the currently open File. I pressed Enter out of habit again because confirmation dialogs are usually a nuisance (as long as you don't make mistakes). :sweat_smile: 

### Recommended actions

- [ ] Select next field by pressing `TAB` once in stead of twice.
- [ ] Do not offer to delete file when pressing `DEL` in Global Search mode.

### Reproducing

1. Press `Ctrl` + `Shift` + `F`
2. Press `Del`
3. Observe file deletion dialog

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.5

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Linux Mint 20 AKA Ubuntu 20.04

**Comment by nathanlesage** [comment]
For half of this issue I now implemented a solution, for the other half I'm missing a keyboard with a physical `Del` key right now

**Comment by nathanlesage** [comment]
Okay, I can not at all reproduce this on macOS, so I think it's specific to Linux (and possibly also Windows)

**Comment by Redsandro** [comment]
> Okay, I can not at all reproduce this on macOS, so I think it's specific to Linux (and possibly also Windows)

Interesting! I just tried it again to make sure I'm not crazy.

1. `Ctrl` + `Shift` + `F`
1. `test`
1. `TAB`
1. `DEL`
1. ![image](https://user-images.githubusercontent.com/1702193/171002116-95eefd14-3bba-44ce-a4d3-07f540c1b04b.png)

**Comment by nathanlesage** [comment]
Yes, I think the DEL keys on macOS behave slightly differently than on Linux and likely also Windows. That was what I wanted to say :D 

I'll boot up my Windows computer tonight after work and see if I can reproduce the behavior!

**Comment by nathanlesage** [comment]
Quick update: Would anyone miss not being able to delete a note just by pressing Delete? If so I'm just going to nuke that shortcut, b/c I think the menu item itself should provide enough safeguards against accidentally moving a file to bin.

If no one objects, tomorrow the shortcut will be gone and thereby this problem.

**Comment by Redsandro** [comment]
No objection from me. I think it's a good idea to remove the shortcut.
